src/main.rs

```diff
@@ -360,11 +360,9 @@ async fn main() {
 
     let app_state = AppState { database: db_data };
 
-    let app = Router::new()
+    let routes = Router::new()
         .route("/:uuid", get(return_config_file_get))
-        .route("/", get(browser_html))
-        .route_service("/style.css", ServeFile::new("static/style.css"))
-        .route_service("/script.js", ServeFile::new("static/script.js"))
+        .nest_service("/migration", axum::routing::get_service(ServeDir::new("static/")),)
```

Comment:
Having this at not the default path is obviously not great. Admittedly it does seem pretty difficult to achieve in axum. Here's how I got it to work with the fallback service.
```rust
        .route("/", axum::routing::get_service(ServeDir::new("static/")))
        .fallback_service(ServeDir::new("static/"))
```
Now that would clash with
```rust
        .route("/:uuid", get(return_config_file_get))
```
but I think it's better to move this to a different sub path anyway and then it works.